frontend/src/components/pages/CampCreation/StepperNavigation/index.tsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import { Box, Center } from "@chakra-ui/react";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import {
+  faCampground,
+  faCalendarDays,
+  faCircleInfo,
+} from "@fortawesome/free-solid-svg-icons";
+import StepperTab from "./StepperTab";
+
+export enum CampCreationPages {
+  CampCreationDetailsPage,
+  ScheduleSessionsPage,
+  RegistrationFormPage,
+}
+
+type StepperProps = {
+  currentPage: CampCreationPages;
+  setCurrentPage: React.Dispatch<React.SetStateAction<CampCreationPages>>;
+  isCampDetailsFilled: boolean;
+  isScheduleSessionsFilled: boolean;
+  isRegistrationFormFilled: boolean;
+};
+
+const Stepper = ({
+  currentPage,
+  setCurrentPage,
+  isCampDetailsFilled,
+  isScheduleSessionsFilled,
+  isRegistrationFormFilled,
+}: StepperProps): React.ReactElement => {
+  return (
+    <Box bg="white" maxW="100%" boxShadow="sm">
+      <Center>
+        <StepperTab
+          title="Camp Details"
+          stepNum={1}
+          focused={currentPage === CampCreationPages.CampCreationDetailsPage}
+          icon={<FontAwesomeIcon icon={faCampground} />}
+          onClick={() => setCurrentPage(CampCreationPages.CampCreationDetailsPage)}
+        />
+        <StepperTab
+          title="Schedule Sessions"
+          stepNum={2}
+          focused={currentPage === CampCreationPages.ScheduleSessionsPage}
+          icon={<FontAwesomeIcon icon={faCalendarDays} />}
+          onClick={() =>
```

Comment:
this is great but we might want to consider disabling this button if the previous step(s) are not filled. Something to differentiate between future and past steps would be useful imo. Waiting on design suggestions